indico/queries/usermetrics.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+from indico.client.request import (
+    GraphQLRequest,
+    PagedRequest,
+)
+from indico.types import BaseType
+from indico.types.user_metrics import UserSummary, UserSnapshot
+
+
+class _PagedUserSnapshots(BaseType):
+    """
+    Class to hold paged snapshot data to make parsing easier.
+    """
+    results: List[UserSnapshot]
+
+
+class _UserMetricsFilters(BaseType):
```

Comment:
this is ok - but doesn't really encompass the full capabilities of the `filter` kwarg. See SubmissionFilter - basically would allow folks to build composite filters as part of their query